setup.py

```diff
@@ -53,6 +52,7 @@
         'tornado': ['tornado >= 4.3'],
         'twisted': ['twisted'],
         'zookeeper': ['kazoo'],
+        'etcd': ['etcd3', 'protobuf==3.20.0'],
```

Comment:
Is it really necessary to pin protobuf to a specific release? Why?

Reply:
This may be because the protobuf [package](https://pypi.org/project/protobuf/) might be installed without your involvement. Etcd3 has a dependency on protobuf. Using the latest protobuf requires setting additional environment variables，and according to the error description, this might impact your program performance.  Someone has the same [issue](https://github.com/kragniz/python-etcd3/issues/2242) in the official repository of python-etcd3.

Reply:
This may be because the protobuf [package](https://pypi.org/project/protobuf/) might be installed without your involvement. Etcd3 has a dependency on protobuf. Using the latest protobuf requires setting additional environment variables，and according to the error description, this might impact your program performance.  Someone has the same [issue](https://github.com/kragniz/python-etcd3/issues/2242) in the official repository of python-etcd3.

Reply:
The problem seems to be that the `python-etcd3` project has not received any commits in a year, and fixing this would just require regenerating the stubs with a newer protobuf release.

Anyway, why not just pin it to `protobuf < 4`? Is there something wrong with the latest 3.x release (3.20.3)?

Reply:
Yeah，you are right. Just 3.20.x or lower. I have changed it.